samples/nrf9160/secure_services/README.rst

```diff
@@ -37,6 +37,6 @@ Building and running
 Dependencies
 ************
 
-This sample uses the following |NCS| libraries:
+This sample uses the following secure firmware component:
 
-* :ref:`lib_spm`
+* :ref:`Trusted Firmware-M <ug_tfm>`
```

Comment:
@oyvindronningstad : Is this true? Does the secure_services sample support TFM, by default. I could see the mention about this in line 20. But the sample describes about SPM. Could you please confirm this.

Reply:
No, the secure_services sample is SPM-only, because of API differences.

Reply:
This is an SPM sample.